roles/idrac_import_server_config_profile/molecule/negative_scenarios-10581/converge.yml

```
@@ -3,26 +3,27 @@
   hosts: all
   gather_facts: false
   tasks:
-    - name: "Importing SCP without share_name"
-      ansible.builtin.import_role:
-        name: "idrac_import_server_config_profile"
-      vars:
-        hostname: "{{ lookup('env', 'IDRAC_IP') }}"
-        username: "{{ lookup('env', 'IDRAC_USER') }}"
-        password: "{{ lookup('env', 'IDRAC_PASSWORD') }}"
-        validate_certs: false
-        target: ['IDRAC']
-        share_parameters:
-          share_user: "{{ lookup('env', 'http_username') }}"
-          share_password: "{{ lookup('env', 'http_password') }}"
-          scp_file: "{{ lookup('env', 'http_filename') }}"
-      ignore_errors: true
-      register: idrac_import_server_config_profile_status
+  ############### Below snippet is commented because of Defect: ECS02A-206 ###############
+    # - name: "Importing SCP without share_name"
+    #   ansible.builtin.import_role:
```

don't comment the task, let it fail if there's defect.

---

done